docker/Dockerfile

```diff
@@ -6,9 +6,14 @@ RUN yum --enablerepo=epel install -y ocl-icd ocl-icd-devel opencl-headers zip un
 RUN groupadd --gid 1000 commp && \
   useradd --gid 1000 --uid 1000 --create-home --shell /bin/bash commp
 
+RUN yum install sudo -y \
```

Comment:
soooo, is this necessary anywhere else in the process? does the Rust installer use this? If it's not necessary I don't think this is a great idea, it's not good practice in general since it gets any activity in the container closer to the host's uid=0.

Reply:
No its not necessary, I accidentality left it in there while debugging.  Will remove, good catch